client/state/notices/README.md

```diff
@@ -40,11 +40,24 @@ this.props.successNotice(
 
 * `successNotice()`
 * `errorNotice()`
+* `infoNotice()`
+* `warningNotice()`
 
 ## Options
 
 The first argument is the text to be displayed on the notice. The second argument is an optional object and accepts some properties:
 
-* `duration: 5000`: (optional) Duration in milliseconds to display the notice before dismissing.
-* `showDismiss: false`: (optional) To indicate if dismiss button should be rendered within the overlay.
+* `id`: ID for your notice
+* `duration: 5000`: (optional, default null) Duration in milliseconds to display the notice before dismissing.
```

Comment:
Would it be possible to blend in the default information to the option description?

For example: 
```
`duration: 5000`: (optional) Time in milliseconds before notice removes itself from page. If duration is not specified, notices will stay on page forever.
```